Kudu.Core/Infrastructure/VsHelper.cs

```diff
@@ -293,11 +239,71 @@ where PathUtilityFactory.Instance.PathsEquals(p.AbsolutePath, targetPath)
                     select p).Any();
         }
 
+        public static string GetTargetFrameworkJson(string path)
+        {
+            try
+            {
+                string document = File.ReadAllText(path);
+                return GetTargetFrameworkJsonContents(document);
+            }
+            catch (Exception)
+            {
+                return string.Empty;
+            }
+        }
+
+        public static string GetTargetFrameworkJsonContents(string content)
```

Comment:
https://docs.microsoft.com/en-us/dotnet/core/tools/project-json-to-csproj#one-target-framework